src/job/notify-open-mrs.job.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-unused-vars */
+import { BotkitMessage } from 'botkit';
+import * as moment from 'moment';
+
+import logger from '../helpers/Logger';
+import { MergeRequest, IChannelMergeRequests } from '../api/mongo';
+import { service as slack, factory as slackFactory } from '../api/slack';
+/* eslint-enable no-unused-vars */
+
+const { NOTIFY_OPEN_MRS_CRON } = process.env;
+const NOTIFY_OPEN_MRS_DELAYED_IN_HOURS = Number(process.env.NOTIFY_OPEN_MRS_DELAYED_IN_HOURS);
```

Comment:
Colocar essa variável dentro da desestruturação acima.

Reply:
Mas eu preciso converter pra `Number`

Reply:
Sim, mas tu tá desestruturando um objeto e na linha de baixo tu já usa uma variável dele que não está na desestruturação atribuindo ela a uma variável com o mesmo nome dando um cast pra Number que tu tá usando só num lugar. Eu desestruturaria ela ali junto e dava o cast só onde eu to usando.

Reply:
Justo, vou ajustar

Reply:
Meu deus, tornei o repositório do meu tcc privado, com medo de entrar e ver comentários do Fumegalli até lá.